packages/daisyui/src/components/tab.css

```diff
@@ -494,15 +494,15 @@
       using --tabs-box-radius CSS variable to avoid Nuxt build issue
       https://github.com/saadeghi/daisyui/issues/3724
     */
-    --tabs-box-radius: calc(var(--radius-field) + var(--radius-field) + var(--radius-field));
+    --tabs-box-radius: calc(3 * var(--radius-field));
```

Comment:
Was there a specific reason why this was written as a sum?
I understand why it was extracted in a variable

Reply:
This line evolved to this slowly. 
Originally it was in a min() in a another calc() and some PostCSS build tools were merging them all, resulting wrong values.  
https://github.com/saadeghi/daisyui/commit/ac358b7ebf7747553cf5fb53ddb718cfb585abba

I moved it to the variable as it was, But now it can indeed be written simpler 👍